site/cypress/e2e/Cypress-Feature/test_forum.spec.js

```diff
@@ -46,6 +47,18 @@ const removeThread = (title) => {
     cy.get('.thread-left-cont > .thread-list-item').contains(title).should('not.exist');
 };
 
+const uploadAttachmentAndDelete = (title) => {
+    cy.get('[data-testid="thread-list-item"]').contains(title).click();
+    cy.get('[data-testid="create-post-head"]').should('contain', title);
+    cy.get('[data-testid="edit-post-button"]').first().click();
+    cy.get('[data-testid="input-file1"]').selectFile('cypress/fixtures/sea_animals.png');
+    cy.get('[data-testid="file-upload-table-1"]').should('contain', 'sea_animals.png');
+    cy.get('[data-testid="forum-update-post"]').contains('Update Post').click();
+    cy.get('[data-testid="edit-post-button"]').first().click();
+    cy.get('[data-testid="mark-for-delete-btn"]').should('contain', 'Delete').click();
+    cy.get('[data-testid="mark-for-delete-btn"]').should('contain', 'Keep');
+    cy.get('[data-testid="forum-update-post"]').contains('Update Post').click();
```

Comment:
I understand the reason for the delete and keep part, it is line 60 that I am uncertain about. Are you asserting that the element contains 'Update Post' or trying to find an element with that testid that has 'Update Post' somewhere in it? 

If you are asserting that it has the value, it would be better to use the should assertion, if not, this is fine. 
```suggestion
    cy.get('[data-testid="forum-update-post"]').contains('Update Post').click();
```

Reply:
used it for finding the `Update Post` there are multiple element with same id
</br>
<img width="1266" alt="Screenshot 2024-05-31 at 11 15 03 PM" src="https://github.com/Submitty/Submitty/assets/119070053/42ca0646-d184-4d16-88e3-209421d5b67e">